### PR TITLE
Update mailing list start page

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/fJCrZY9u/7054-update-content-on-mailing-list-start-page

### CODE REVIEW

File: docs/sign-up-journeys.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ There are multiple different sign up journeys available on the Get into Teaching
 
 ```mermaid
 graph TD;
-  name[Get personalised guidance on teacher training in your inbox] -- Doesn't exist in CRM --> returning_teacher[Are you already qualified to teach?]
+  name[Free personalised teacher training guidanceer training in your inbox] -- Doesn't exist in CRM --> returning_teacher[Are you already qualified to teach?]
```
Comment (emmaachesongray) [inline docs/sign-up-journeys.md:9]:
```suggestion
  name[Free personalised teacher training guidance] -- Doesn't exist in CRM --> returning_teacher[Are you already qualified to teach?]
```

File: spec/features/mailing_list_wizard_spec.rb [hunk 2]
```diff
@@ -135,7 +135,7 @@
 
     visit mailing_list_steps_path({ id: :name, channel: "invalid", sub_channel: sub_channel_id })
 
-    expect(page).to have_text "Get personalised guidance on teacher training in your inbox"
+    expect(page).to have_text "Free personalised teacher training guidancee"
```
Comment (emmaachesongray) [inline spec/features/mailing_list_wizard_spec.rb:138]:
```suggestion
    expect(page).to have_text "Free personalised teacher training guidance"
```

File: app/views/mailing_list/steps/_name.html.erb [hunk 3]
```diff
@@ -1,27 +1,14 @@
-<h1>Get personalised guidance on teacher training in your inbox</h1>
+<h1>Free personalised teacher training guidance</h1>
 
 <p>
-If you're interested in training to teach in primary, secondary and special schools in England, get everything you need to know to start a career in teaching.
+Discover everything you need to know to start a teaching career in England, sent straight to your inbox. Personalised to you, we'll send you the latest information as well as advice and support.
```
Comment (gemmadallmandfe) [inline app/views/mailing_list/steps/_name.html.erb:4]: We say `personalised` in the h1 and in the text here - little bit repetitive or reinforcing the message?

File: app/views/mailing_list/steps/_name.html.erb [hunk 4]
```diff
@@ -1,27 +1,14 @@
...
 <%= f.govuk_text_field :last_name, width: 'two-thirds', autocomplete: "family-name" %>
 <%= f.govuk_email_field :email, width: 'two-thirds', autocomplete: "email" %>
 <%= f.hidden_field :channel_id, value: params[:channel] || f.object.channel_id %>
 <%= f.hidden_field :sub_channel_id, value: params[:sub_channel] || f.object.sub_channel_id %>
 <%= f.hidden_field :accepted_policy_id, value: f.object.latest_privacy_policy.id %>
 
-<p>Your details are protected under the terms of our <%= link_to("privacy notice (opens in new tab)", privacy_policy_path(id: f.object.latest_privacy_policy.id), { target: :blank }) %>.</p>
-
-<p>Our privacy notice explains how we use your personal data. It's important you read this before signing up to receive emails.</p>
+<p>Your details are protected under the terms of our <%= link_to("privacy notice (opens in new tab)", privacy_policy_path(id: f.object.latest_privacy_policy.id), { target: :blank }) %>. Our privacy notice explains how we use your personal data. It's important you read this before signing up to receive emails.</p>
```
Comment (gemmadallmandfe) [inline app/views/mailing_list/steps/_name.html.erb:14]:
```suggestion
<p>Your details are protected under the terms of our <%= link_to("privacy notice (opens in new tab)", privacy_policy_path(id: f.object.latest_privacy_policy.id), { target: :blank }) %>. This explains how we use your personal data. It's important you read this before signing up to receive emails.</p>
```

Wondering if we can shorten the wording slightly if it is combined into one paragraph?

But needs @GayeCord's approval re both of these things!

Comment (GayeCord) [reply app/views/mailing_list/steps/_name.html.erb:14]: As long as we are giving people a clear link to the PN, i'm happy with however you want to wrap content around it

File: app/views/mailing_list/steps/_name.html.erb (hunk 3 above)
Comment (emmaachesongray) [reply app/views/mailing_list/steps/_name.html.erb:4]: I think it reinforces the message that it's not just a generic mailing list

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Pull request merged**